internal/cmd/asyncapi/command_import.go

```diff
@@ -0,0 +1,443 @@
+				SchemaType: resolveSchemaType(components.Messages[strcase.ToCamel(topicName)+"Message"].ContentType),
+			})
+		if err != nil {
+			return 0, fmt.Errorf("unable to register schema: %v", err)
+		}
+		output.Printf("Schema registered under subject %s with ID %d.\n", subject, id.Id)
+		return id.Id, nil
+	}
+	return 0, fmt.Errorf("schema payload not found in YAML input file")
+}
+
+func updateSubjectCompatibility(details *accountDetails, compatibility string, subject string) error {
+	// Updating the subject level compatibility
+	log.CliLogger.Infof("Updating the Subject level compatibility to %s", compatibility)
+	updateReq := srsdk.ConfigUpdateRequest{Compatibility: compatibility}
+	config, _, err := details.srClient.DefaultApi.UpdateSubjectLevelConfig(details.srContext, subject, updateReq)
+	if err != nil {
+		return fmt.Errorf("failed to update subject level compatibility: %v", err)
+	}
+	log.CliLogger.Infof("Subject level compatibility updated to %s", config.Compatibility)
+	return nil
+}
+
+func addSchemaTags(details *accountDetails, components Components, topicName string, schemaId int32) error {
+	// Schema level tags
+	var tagConfigs []srsdk.Tag
+	var tagDefConfigs []srsdk.TagDef
+	if components.Messages != nil {
+		if components.Messages[strcase.ToCamel(topicName)+"Message"].Tags == nil {
+			return nil
+		}
+		for _, tag := range components.Messages[strcase.ToCamel(topicName)+"Message"].Tags {
+			tagDefConfigs = append(tagDefConfigs, srsdk.TagDef{
+				// tag of type cf_entity so that it can be attached at any topic or schema level
+				EntityTypes: []string{"cf_entity"},
+				Name:        tag.Name,
+			})
+			tagConfigs = append(tagConfigs, srsdk.Tag{
+				TypeName:   tag.Name,
+				EntityType: "sr_schema",
+				EntityName: fmt.Sprintf("%s:.:%s", details.srCluster.Id, strconv.Itoa(int(schemaId))),
+			})
+		}
+		err := addTagsUtil(details, tagDefConfigs, tagConfigs)
+		if err != nil {
+			return err
+		}
+		output.Printf("Tags added to schema %s\n", strconv.Itoa(int(schemaId)))
+	}
+	return nil
+}
+
+func addTopicTags(details *accountDetails, subscribe Operation, topicName string) error {
+	// Topic level tags
+	// add topic level tags only if topic was created successfully or already exists
+	if subscribe.TopicTags == nil {
+		return nil
+	}
+	tagConfigs := []srsdk.Tag{}
+	tagDefConfigs := []srsdk.TagDef{}
+	for _, tag := range subscribe.TopicTags {
+		tagDefConfigs = append(tagDefConfigs, srsdk.TagDef{
+			// tag of type cf_entity so that it can be attached at any topic or schema level
+			EntityTypes: []string{"cf_entity"},
+			Name:        tag.Name,
+		})
+		tagConfigs = append(tagConfigs, srsdk.Tag{
+			TypeName:   tag.Name,
+			EntityType: "kafka_topic",
+			EntityName: details.srCluster.Id + ":" + details.clusterId + ":" + topicName,
+		})
+	}
+	err := addTagsUtil(details, tagDefConfigs, tagConfigs)
+	if err != nil {
+		return err
+	}
+	output.Printf("Tags added to Kafka topic %s\n", topicName)
```

Comment:
```suggestion
	output.Printf("Tags added to Kafka topic \"%s\".\n", topicName)
```

Reply:
All of your output should be tested with integration tests. I don't see any golden files that contain this output, so I have no way of knowing if this actually works!